Main/lib/perl/WorkflowSteps/InsertInterproResults.pm

```diff
@@ -0,0 +1,33 @@
+package ApiCommonWorkflow::Main::WorkflowSteps::InsertInterproResults;
+
+@ISA = (ApiCommonWorkflow::Main::WorkflowSteps::WorkflowStep);
+
+use strict;
+use ApiCommonWorkflow::Main::WorkflowSteps::WorkflowStep;
+
+sub run {
+    my ($self, $test, $undo) = @_;
+
+    my $resultsFile = $self->getParamValue('resultsFile');
+    my $transcriptSourceId = $self->getParamValue('transcriptSourceId');
+    my $geneSourceId = $self->getParamValue('geneSourceId');
+    my $ncbiTaxId = $self->getParamValue('ncbiTaxId');
+
+    my $workflowDataDir = $self->getWorkflowDataDir();
+  
+    my $args = <<"EOF";
+--resultsFile=$workflowDataDir/$resultsFile \\
+--transcriptSourceId=$transcriptSourceId \\
```

Comment:
i don't understand these 2 params

Reply:
The transcriptSourceID is no longer needed. The resultsFile is the result file output from the running of interproscan.